stat/distuv/binomial.go

```diff
@@ -74,16 +74,16 @@ func (b Binomial) Prob(x float64) float64 {
 // Rand returns a random sample drawn from the distribution.
 func (b Binomial) Rand() float64 {
 	// NUMERICAL RECIPES IN C: THE ART OF SCIENTIFIC COMPUTING (ISBN 0-521-43108-5)
-	// p. 294
+	// p. 295-6
 	// <http://www.aip.de/groups/soe/local/numres/bookcpdf/c7-3.pdf>
 
-	unifrnd := rand.Float64
-	exprnd := rand.ExpFloat64
+	runif := rand.Float64
+	rexp := rand.ExpFloat64
 	var rng *rand.Rand
```

Comment:
`rng` is not used outside the conditional scope below, so this does not need to exist and you can say `rng := rand.New(b.Src)`, though we usually spell it `rnd`.